src/o365/graph/commands/subscriptions/subscription-add.ts

```diff
@@ -0,0 +1,244 @@
+import commands from '../../commands';
```

Comment:
Folder for subscription-related commands should be named `subscription` rather than `subscriptions`